framework/src/sbt-plugin/src/sbt-test/shutdown/downing/project/Build.scala

```diff
@@ -84,4 +86,44 @@ object DevModeBuild {
         }
     }
   }
+
+
+  val assertProcessIsStopped: Command = Command.args("assertProcessIsStopped", "") { (state: State, args: Seq[String]) =>
+    val pidFile = Project.extract(state).get(Keys.target) / "universal" / "stage" / "RUNNING_PID"
+    if(!pidFile.exists())
+      throw new RuntimeException("RUNNING_PID file not found. Can't assert the process is stopped without knowing the process ID.")
+
+    val pidString = Files.readAllLines(pidFile.getAbsoluteFile.toPath).get(0)
+
+    def processIsRunning(pidString: String): Boolean ={
+      val foundProcesses = Common.runProcess("jps") // runs the command and returns the output as a single String.
+        .split("\n") // split per line
+        .filter{_.contains("ProdServerStart")}
+      foundProcesses // filter only the Play processes
+        // This assertion is flaky since `11234` contains `123`. TODO: improve matcher
+        .exists(_.contains(pidString)) // see if one of them is PID
```

Comment:
```suggestion
        .exists(_.startsWith(pidString + " ")) 
```

It's hacky, but simple and efficient. :-)